services/director-v2/tests/unit/with_dbs/test_route_clusters_details.py

```diff
@@ -176,6 +176,10 @@ async def _get_cluster_details(
     return cluster_out
 
 
+# NOTE: ANE -> SAN on my local computer it prints out:
+# AssertionError: worker did not update the cpu metrics
+# assert 4.0 == 0
```

Comment:
yes I saw that this one sometimes fail. does it systematically fail for you?

Reply:
yes, I have it failing in the CI also locally

Reply:
so this one is fixed in master. can you please revert the file?